test/ProjectService.Business.UnitTests/Commands/CreateNewProjectCommandTests.cs

```diff
@@ -90,22 +100,50 @@ public void ShouldThrowsExceptionWhenRepositoryThrowsException()
         }
 
         [Test]
-        public void ShouldReturnIdWhenCreatingNewProject()
+        public void ShouldReturnIdWhenCreatingNewProjectAndUserIsAdmin()
         {
-            validatorMock
-                 .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
-                 .Returns(true);
+            accessValidator
+                .Setup(x => x.HasRights(It.IsAny<int>()))
+                .Returns(false);
+            Assert.AreEqual(newProject.Id, command.Execute(newRequest));
+            mapperMock.Verify();
+            repositoryMock.Verify();
```

Comment:
this not checking because setup method have not `Verifable()`

Reply:
Fixed.